interfaces/builtin/optical_drive.go

```diff
@@ -46,13 +46,15 @@ const opticalDriveConnectedPlugAppArmor = `
 # Allow read access to optical drives
 /dev/sr[0-9]* r,
 /dev/scd[0-9]* r,
+/dev/sg[0-9]* r,
 @{PROC}/sys/dev/cdrom/info r,
 /run/udev/data/b11:[0-9]* r,
 `
 
 var opticalDriveConnectedPlugUDev = []string{
 	`KERNEL=="sr[0-9]*"`,
 	`KERNEL=="scd[0-9]*"`,
+	`SUBSYSTEM=="scsi_generic", SUBSYSTEMS=="scsi", ATTRS{type}=="4|5"`,
```

Comment:
Might be nice to have a comment somewhere what the values of "4|5" means in this context.

Reply:
I plucked those out of the default rules file in `/lib/udev/rules.d/50-udev-default.rules` from the rule that assigns the group of `cdrom` to appropriate devices. According to https://en.wikipedia.org/wiki/SCSI_Peripheral_Device_Type the types are:

4. Write-once device
5. CD/DVD-ROM device

I can add those definitions into the source as a comment if it would help.

Reply:
We might get away with dropping type `4`?

Reply:
Thanks for adding this comment !